sparkbox/bouncy-ball
# add: new example using Lottie / Bodymovin

**To test this locally:**
1. Run `npm run build`
2. Run `npm start` and open the site at local url provided
3. Confirm that
    1. "Lottie" was added to the list
    2. The animation is working
    3. The docs are displayed when you click "More about Lottie"
